src/CraneCtld/CtldGrpcServer.cpp

```
@@ -93,29 +104,79 @@ grpc::Status CraneCtldServiceImpl::TaskStatusChange(
 }
 
+grpc::Status CraneCtldServiceImpl::CranedTriggerReverseConn(
+    grpc::ServerContext *context,
+    const crane::grpc::CranedTriggerReserveConnRequest *request,
+    google::protobuf::Empty *response) {
+  const auto &craned_id = request->craned_id();
+  CRANE_TRACE("Craned {} requires Ctld to connect.", craned_id);
+  if (!g_meta_container->CheckCranedAllowed(request->craned_id())) {
+    CRANE_WARN("Reject register request from unknown node {}",
+               request->craned_id());
+    return grpc::Status::OK;
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)